include/CppUTestExt/MockNamedValue.h

```diff
@@ -66,7 +66,7 @@ class MockFunctionComparator : public MockNamedValueComparator
  * Basically this class ties together a Name, a Value, a Type, and a Comparator
```

Comment:
You'll want the correction that @jgonzalezdr did to the comment on line 31 as well